### ISSUE TITLE
Support for Better Safe Bed

### ISSUE THREAD

**Issue by NebelNidas** [Minenash/MonstersInTheCloset#2]
Would it be possible to add compatibility with [Better Safe Bed](https://www.curseforge.com/minecraft/mc-mods/better-safe-bed)? That mod's developer says that he can't do that himself because of some mixins, perhaps it would be possible from your side?

**Comment by Minenash** [comment]
Funnily enough, in the version I published earlier this week, I removed the redundant check, and replaced it with "Is the fail reason `NOT_SAFE`?". This is how it should've always been, but I didn't know how to properly mixin a lambda when I originally wrote the mod.

So in theory the latest version should already work together.

~~Edit: Though I am slightly confused about how their mod actually works. Like I know it does (just tried it), but looking at the vanilla code and his, I don't get what they're changing that the vanilla doesn't already does. In vanilla, it gets a list of only HostileEntites, so removing non-HostileEntities seems redundant and their checks if it's target is a player, but vanilla only includes the HostileEntity if it's angry at the player trying to sleep.~~
